docs/docs/building-modules/01-module-manager.md

```diff
@@ -24,18 +24,18 @@ It is recommended to implement interfaces from the [Core API](https://docs.cosmo
 For legacy reason modules can still implement interfaces from the SDK `module` package.
 :::
 
-There are 2 main application module interfaces:
+There are 3 main application module interfaces:
 
 * [`appmodule.AppModule` / `module.AppModule`](#appmodule) for inter-dependent module functionalities (except genesis-related functionalities).
 * (legacy) [`module.AppModuleBasic`](#appmodulebasic) for independent module functionalities. New modules can use `module.CoreAppModuleBasicAdaptor` instead.
+* (legacy) `module.GenesisOnlyAppModule` defines an `AppModule` that only has import/export functionality.
```

Comment:
this isnt really a main module interface. It reads as though this must be implemented if its a main interface.

Reply:
Right, this isn't even an interface, so it has nothing to do here :D